velox/type/Conversions.h

```diff
@@ -418,6 +455,51 @@ struct Converter<TypeKind::VARCHAR, void, TRUNCATE, LEGACY_CAST> {
   static std::string cast(const bool& val) {
     return val ? "true" : "false";
   }
+
+  static void normalizeStandardNotation(std::string& str) {
+    if (!FLAGS_experimental_enable_legacy_cast &&
```

Comment:
Do we need both LEGACY_CAST template parameter and FLAGS_experimental_enable_legacy_cast gflag?

Reply:
I think we can combine these two flags and only use LEGACY_CAST.

Reply:
For this flag I once checked with @bikramSingh91 and it's being set by xstream. However, xstream is in progress deprecating it now, so we can deprecate it very soon. So it's from history.